features/networking/multus.feature

```diff
@@ -1263,3 +1263,28 @@ Feature: Multus-CNI related scenarios
       | /usr/sbin/ip | route         |
     Then the output should contain:
       | 192.168.10.0                 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-30054
+  @admin
+  Scenario: Multus namespaceIsolation should allow references to CRD in the default namespace
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    # Create the net-attach-def via cluster admin
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-macvlan.yaml" replacing paths:
+      | ["metadata"]["namespace"] | default                                                                                                                         |
+      | ["spec"]["config"]        | '{ "cniVersion": "0.3.0", "type": "macvlan", "mode": "bridge", "ipam": { "type": "whereabouts", "range": "192.168.22.100/24"} }'|
+    Then the step should succeed
+
+    #Cleanup created net-attach-def from default namespaces
+    And admin ensures "macvlan-bridge-whereabouts" network_attachment_definition is deleted from the "default" project after scenario
+    
+    # Create a pod absorbing above net-attach-def
+    Given I switch to the first user
```

Comment:
@weliang1 lgtm except a little nit-bit, since we have removed the pseudo step, we don't need to switch to first user as this whole scenario is now being executed by first user except net-attach def creation by admin.
Also if you can expand comment ```Create a pod absorbing above net-attach-def``` to may be like ``` Create a pod absorbing above net-attach-def wit reference to default namespace``` would be great for understanding. Thanks!

Reply:
@anuragthehatter Agree, fixed them based on above two comments. Thanks!

Reply:
Re run the script: https://privatebin-it-iso.int.open.paas.redhat.com/?668d2b2e65bee184#r8wS7SPsLJZpMHiZWLja0pCx5JVK25yD2N63h9svvX4=